Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Based off https://medium.com/@chemidy/create-the-smallest-and-secured-golang-docker-image-based-on-scratch-4752223b7324
+
+# Accept the Go version for the image to be set as a build argument.
+ARG GO_VERSION=1.11.2
+
+# First stage: build the executable.
+FROM golang:${GO_VERSION}-alpine AS builder
+
+# ca-certificates for calls to HTTPS endpoints.
+# git for fetching the dependencies.
+RUN apk add --no-cache \
+	ca-certificates \
+	git
+
+# Create appuser
+RUN adduser -D -g '' appuser
+
+# Get code
+COPY . $GOPATH/src/github.com/peterbourgon/fastly-exporter/
+WORKDIR $GOPATH/src/github.com/peterbourgon/fastly-exporter/
+
+# Get dependancies
+RUN go get -d -v
```

Comment:
Since we vendor deps and check in the vendor folder, I guess this should be deleted.